README.md

```diff
@@ -126,6 +126,110 @@ const Moltin = MoltinGateway({
 })
 
 ``` 
+### TypeScript Support
+The Moltin SDK is fully supported in Typescript. Each service is wrapped in a namespace which exposes all the necessary interfaces and endpoints that can be consumed.
+
+Each response will likely be returned as either a `Response` (single object returned) or a `ResponsePage` (an array of items) - see `src/types/core.d.ts` for more info.
+
+Current exposed namespaces include `address`, `brand`, `cart`, `category`, `collectoin`, `core`, `currencies`, `customer`, `field`, `flow`, `inergration`, `inventory`, `job`, `order`, `product` and `transaction`. For further information on any of these services, please refer to the [docs](https://documentation.elasticpath.com/commerce-cloud/docs/api).
```

Comment:
`collectoin`

Reply:
`inergration`

Reply:
While we're updating, change `docs` to `documentation`